README.md

```diff
@@ -35,7 +40,83 @@ cd saliency
 
 ## Usage
 
-Each saliency mask class extends from the `SaliencyMask` base class. This class
+The saliency library has two subpackages:
+*	`core` uses a generic `call_model_function` which can be used with any ML 
+	framework.
+*	`tf1` accepts input/output tensors directly, and sets up the necessary 
+	graph operations for each method.
+
+### Core
+
+Each saliency mask class extends from the `CoreSaliency` base class. This class
+contains the following methods:
+
+*   `GetMask(x_value, call_model_function, call_model_args)`: Returns a mask of
+    the shape of non-batched `x_value` given by the saliency technique.
+*   `GetSmoothedMask(x_value, call_model_function, call_model_args)`: Returns a
```

Comment:
We can add those in, although they aren't required since they have defaults. Should we add in the defaults as part of documentation? i.e:

GetSmoothedMask(self, x_value, call_model_function, call_model_args=None, stdev_spread=.15, nsamples=25, magnitude=True)

Reply:
Added in optional arguments here with their default parameters, let me know what you think.